core/domain/activity_jobs_one_off.py

```diff
@@ -24,16 +24,9 @@
 from core.platform import models
 import feconf
 
-(
-    collection_models, config_models,
-    exp_models, question_models,
-    skill_models, story_models,
-    topic_models) = (
-        models.Registry.import_models([
-            models.NAMES.collection, models.NAMES.config,
-            models.NAMES.exploration, models.NAMES.question,
-            models.NAMES.skill, models.NAMES.story,
-            models.NAMES.topic]))
+(collection_models, exp_models, topic_models) = (
```

Comment:
Shouldn't this have been flagged by the linter as variable not used or something like that?

Reply:
I *think* it will detect regular imports, but not this sort of import. It would be nice to add it as a check though. I've added it to the list in #8423.